continuous-testing/starter-kits/README.md

```diff
@@ -1,3 +1,54 @@
 # Starter Kits
 
-Nothing here yet, come back soon... 
\ No newline at end of file
+This page contains starter kit information, which represent templates, code and configuration to help you get started quickly with continuous testing best practices described in this overall guide. Please see categories and links below for details. 
+
+## Software Composition Analysis
+
+This section contains links to sample actions, templates and configurations that analyze and validate composition of Open Source Software (OSS) components in software systems. Identifying software and licensing vulnerabilites and ensuring routine software updates is an [OSS cybersecurity](https://www.cisa.gov/uscert/ncas/alerts/aa22-137a) [best practice](https://appel.nasa.gov/2022/06/30/spotlight-on-lessons-learned-open-source-and-commercial-web-software-vulnerabilities/).  
+
+### Dependabot
+
+#### Automated Dependency Updates
+
+This Dependabot task provides an automated check to test OSS component versions and automatically create [pull requests](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/managing-pull-requests-for-dependency-updates) to update components. 
+
+Starter Kit:
+- [SLIM Starterkit Python -- Dependabot Script](https://github.com/NASA-AMMOS/slim-starterkit-python/blob/develop/.github/dependabot.yml) to install in your GitHub repo
+
+To leverage this template, make sure to do the following:
+1. Discuss with your development team the cybersecurity best practice to regularly update OSS to latest versions and seek consensus on a workflow to accept proposed updates.
+2. Add Dependabot automation to your repository, either **manually** or via the **admin console** (choose **one**):
+> **Shortcut**  
+  Copy the pre-set configuration to an identical path in your repository, e.g. `.github/dependabot.yml`.
+   * **Manually:** (_recommended approach_ available to all committers)
+      1. Create an issue and an issue branch to implement a code change. Checkout the issue branch.
+      2. Copy the Dependabot configuration file from one of the SLIM Starterkit repos -- for example, [`dependabot.yml` in the Python Starterkit](https://github.com/NASA-AMMOS/slim-starterkit-python/blob/main/.github/dependabot.yml) -- into the root of your repository at `.github/dependabot.yml`.
+   * **Admin console:** (requires admin rights)
```

Comment:
Might be good to put the admin one first since that’s the easiest. If people don’t have admin privileges, then they can go through the manual route.